airflow/utils/logging.py

```diff
@@ -211,3 +212,30 @@ def parse_gcs_url(self, gsurl):
             bucket = parsed_url.netloc
             blob = parsed_url.path.strip('/')
             return (bucket, blob)
+
+
+def get_log_filename(**kwargs):
+    """
+    Return full log filename. For example:
+        dag_id/task_id/execution_date/try_number.log
+    """
+    relative_dir = get_log_directory(**kwargs)
+    try_number = kwargs.get('try_number')
+    if try_number is None:
```

Comment:
Won't need this check once you make try number a required parameter. in the signature.